msvc2017/update_stdafx.py

```diff
@@ -141,18 +147,17 @@ def update_stdafx_h(stdafx_h_filename, header_only_vcxproj_files, vcxproj_files)
 
         lines.append(include_directive)
 
-    with open(stdafx_h_filename, 'w') as f:
+    with open(stdafx_h_filename, "w") as f:
```

Comment:
why is `"` preferred over `'` ?

Reply:
Just to use the same style evrythere. 

https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings